cms/templates/js/certificate-editor.underscore

```diff
@@ -0,0 +1,47 @@
+<form class="collection-edit-form certificate-edit-form">
+    <div aria-live="polite">
+    <% if (error && error.message) { %>
+        <div class="certificate-edit-error message message-status message-status error is-shown" name="certificate-edit-error">
+          <%= gettext(error.message) %>
+        </div>
+    <% } %>
+    </div>
+    <div class="wrapper-form">
+        <fieldset class="collection-fields certificate-fields">
+            <legend class="sr"><%= gettext("Certificate Information") %></legend>
+            <div class="input-wrap field text required add-collection-name add-certificate-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
+                <label for="certificate-name-<%= uniqueId %>"><%= gettext("Certificate Name") %></label>
+                <% if (!_.isUndefined(id)) { %>
+                    <span class="certificate-id">
+                        <span class="certificate-label"><%= gettext('Certificate ID') %></span>
+                        <span class="certificate-value"><%= id %></span>
+                    </span>
+                <% } %>
+                <input id="certificate-name-<%= uniqueId %>" class="collection-name-input input-text" name="certificate-name" type="text" placeholder="<%= gettext("Name of the certificate") %>" value="<%= name %>" aria-describedby="certificate-name-<%=uniqueId %>-tip">
+                <span id="certificate-name-<%= uniqueId %>-tip" class="tip tip-stacked"><%= gettext("Name of the certificate") %></span>
+            </div>
+            <div class="input-wrap field text add-certificate-description">
+                <label for="certificate-description-<%= uniqueId %>"><%= gettext("Description") %></label>
+                <textarea id="certificate-description-<%= uniqueId %>" class="certificate-description-input text input-text" name="certificate-description" placeholder="<%= gettext("Description of the certificate") %>" aria-describedby="certificate-description-<%=uniqueId %>-tip"><%= description %></textarea>
+                <span id="certificate-description-<%= uniqueId %>-tip" class="tip tip-stacked"><%= gettext("Description of the certificate") %></span>
+            </div>
+        </fieldset>
+         <header>
+            <span class="certificate-label"><%= gettext("Certificate Signatories") %></legend>
+         </header>
+        <div class="signatory-edit-list"> </div>
+        <span>
+           <button class="action action-add-signatory" type="button"><%= gettext("Add Signatory") %></button>
```

Comment:
Styles seem to be broken for this button action currently. This should use the same styling as the add / create actions at the bottom of lists, as shown below:

![image](https://cloud.githubusercontent.com/assets/2023680/7681514/dc64e5fc-fd3e-11e4-9c2a-423b40d6a0ea.png)